Laggan-Gang/Claes
# Third-party allegations

## Description
- Implementing the API that both shuffles & assigns roles, depending on user's preferences. 

- Built next to & independent from `maakep happen` so that the maakep happen experience remains authentic. It can easily be built into that one too though, if this PoC turns out to be appreciated.

- I did some reorganizing of the audio clips & formatting of code (but kept the laggan copy-righted comments & swenglish of course). If there are undesired, I will simply omit the commits - no problem.

## How to test

1. Update `config.json` by adding this line:

```
{
       "token": "blablasomething",
--->   "dotaPrefsBaseUrl": "GET ADDRESS FROM ME"    <---
}
```

2. In discord, where bot lives, write & verify that you get expected output:

- `!dota set maakep 4 5 fill 1 2 3`
- `!dota roll maakep mbar jorel `
- `!dota delete maakep`
- `!dota link`

3. You can visit the site to verify the data.

4. ???

5. Profit


## Known issues
- Not in this implementation but the root implementation doesn't fully respect the rules of fill, it will add people last in the queue. So if you're rolled for last and fill, you will still get the last pick, but should actually be first.
